storage/named_query.go

```diff
@@ -39,6 +40,17 @@ var namedQueries = map[NamedQuery]string{
 	(SELECT ID FROM {{.LABELS_TABLE}} 
 				WHERE key=:key
 				AND {{.ENTITY_TABLE}}.{{.PRIMARY_KEY}} = {{.LABELS_TABLE}}.{{.REF_COLUMN}})`,
+	QueryForLastOperationsPerResource: `
```

Comment:
can we make this one look more like the neighbours? CAPITALS, {{.ENTITY_TABLE}} etc.

Reply:
Do you see a use case for having ENTITY_TABLE variable in this statement?

Reply:
operations? but i get your point..